### DISCUSSION TITLE
on mobile, post has a huge slider to the right and  nav bar is not visible

### DISCUSSION THREAD

**Discussion by chwunder** [daattali/beautiful-jekyll#858] [Q&A]
Hi everyone,

if my blog is being used on a phone, but just a blogpost, not the website itself, i get a huge slider to the right. My face in the middle as well as the navigation in the right top corner is not visible anymore, you need to scroll quite a lot!
[https://christianwunderlich.com/2021-08-20-Python-EventHub-ADX-Ingest/](https://christianwunderlich.com/2021-08-20-Python-EventHub-ADX-Ingest/)

If you you take a post from https://beautifuljekyll.com/2020-02-26-flake-it-till-you-make-it/ it doesn't happen. Can someone let me know how to fix this please!

Thanks and best regards,
Chris

**Comment by daattali** [comment]
It's probably caused by the reference table you have on the bottom of the page, because it has URLs that are very long, the URL isn't being broken up to multiple lines since there's no whitespace in it. You can force the browser to break up the long URL into multiple lines by adding the following CSS rule (if you aren't familiar with CSS or you don't know how to add CSS, you'll have to do a little bit of googling for that)

```
a {
    word-break: break-all;
}
```

**Reply by chwunder** [reply]
Thanks buddy, looks good now! appreciate your help!
